stix_shifter_modules/cbcloud/stix_translation/json/to_stix_map.json

```diff
@@ -0,0 +1,180 @@
+{
+  "process_username": [
+    {
+      "key": "user-account.account_login",
```

Comment:
so the `user-account` object also needs `user_id`. as per stix specification because that's a required property. I wonder if you can set it to user_id instead or if you have any other field that can be set as user_id

Reply:
Will change. I was using like wat appeared to be the most appropriate field (a string/word seemed to fit better with account_login than with id, which sounded like it would be more of a int/number). If the stix specification requires at the very least to have 'id' set in a user_account, then I can change. Cb Cloud does not have a user ID field, just process_username.

Reply:
This has been updated together with the equivalent value in the `from_stix_map.json` file.